pom.xml

```diff
@@ -27,6 +27,7 @@
         <module>utils</module>
         <module>model</module>
         <module>validation</module>
```

Comment:
Should the `LoggingFilter` move into this project?

Reply:
Maybe eventually